apps/osvr_server.cpp

```diff
@@ -92,7 +83,7 @@ int main(int argc, char *argv[]) {
     }
 
     if (values.count("help")) {
-        std::cout << optionsVisible << std::endl;
+        log->info() << optionsVisible << std::endl;
```

Comment:
In this particular case, we should use std::cout instead of a logger because we don't want the output lines prefixed with timestamps, etc.